langchain/client/runner_utils.py

```diff
@@ -651,6 +694,7 @@ def run_on_dataset(
     client: Optional[LangChainPlusClient] = None,
     tags: Optional[List[str]] = None,
     run_evaluators: Optional[Sequence[RunEvaluator]] = None,
+    input_mapper: Optional[Callable[[Dict], Any]] = None,
```

Comment:
This is a lot of arguments. Also may as well make the other handlers an input mapper at that point?